property_config.go

```diff
@@ -37,6 +37,27 @@ type NumberPropertyConfig struct {
 	Format FormatType         `json:"format"`
 }
 
+// MarshalJSON returns encoded json.
+// Modify the json structure without changing the fields in NumberPropertyConfig for backward compatibility.
+func (p NumberPropertyConfig) MarshalJSON() ([]byte, error) {
```

Comment:
Let's just deprecate the `Format` field and add the `Number`. I think it's much simpler and explicit
```

type NumberPropertyConfig struct {
	ID     ObjectID           `json:"id,omitempty"`
	Type   PropertyConfigType `json:"type"`
        // DEPRECATED: use Number instead
	Format FormatType         `json:"format"`
	Number struct {
	    Format FormatType `json:"format"`
	} `json:"number"`
}
```

WDYT?

Reply:
Great suggestion, I modified it to a simpler form using the Deprecated comment !
https://github.com/jomei/notionapi/pull/94/commits/14ada7b3aeea0df47df22962e8c808f7180450ba

Reply:
Format required omitempty tag. Otherwise, a "" would have been specified for the unspecified Format, and the Notion API would return a similar error.